framework/tlvf/yaml/tlvf/WSC/eWscVendorExt.yaml

```diff
@@ -5,9 +5,11 @@ _namespace: WSC
 eWscVendorExtSubelementBssType:
   _type: enum
   BACKHAUL_STA: 0x80
+  FRONTHAUL_BACKHAUL_BSS: 0x60 # (FRONTHAUL_BSS | BACKHAUL_BSS)
   BACKHAUL_BSS: 0x40
   FRONTHAUL_BSS: 0x20
   TEARDOWN: 0x10
+  WRONG_TYPE: 0
```

Comment:
IMO we don't need to define a WRONG_TYPE..

Reply:
I can rename it as an Invalid type.  We need it for writing new APIs. @arnout What do you think?